README.md

```diff
@@ -20,10 +20,22 @@ console.log(second_array) // [1, 2, 3]
 
 ```js
 function swap(arr1, arr2) {
-  // -- YOUR ANSWER HERE --
+  var placeholder = arr2;
+  arr2 = arr1;
+  arr1 = placeholder;
```

Comment:
You have the right idea.  The problem is that this does not change the original arrays.